src/components/DropzoneAreaFunc.js

```diff
@@ -0,0 +1,486 @@
+import Snackbar from "@material-ui/core/Snackbar";
+import Typography from "@material-ui/core/Typography";
+import { withStyles } from "@material-ui/core/styles";
+import CloudUploadIcon from "@material-ui/icons/CloudUpload";
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React, { useState, useEffect } from "react";
+import { Fragment } from "react";
+import Dropzone from "react-dropzone";
+
+import {
+  convertBytesToMbsOrKbs,
+  createFileFromUrl,
+  readFile,
+} from "../helpers";
+
+import PreviewList from "./PreviewList";
+import SnackbarContentWrapper from "./SnackbarContentWrapper";
+
+const styles = {
+  "@keyframes progress": {
+    "0%": {
+      backgroundPosition: "0 0",
+    },
+    "100%": {
+      backgroundPosition: "-70px 0",
+    },
+  },
+  dropZone: {
+    position: "relative",
+    width: "100%",
+    minHeight: "250px",
+    backgroundColor: "#F0F0F0",
+    border: "dashed",
+    borderColor: "#C8C8C8",
+    cursor: "pointer",
+    boxSizing: "border-box",
+  },
+  stripes: {
+    border: "solid",
+    backgroundImage:
+      "repeating-linear-gradient(-45deg, #F0F0F0, #F0F0F0 25px, #C8C8C8 25px, #C8C8C8 50px)",
+    animation: "progress 2s linear infinite !important",
+    backgroundSize: "150% 100%",
+  },
+  rejectStripes: {
+    backgroundImage:
+      "repeating-linear-gradient(-45deg, #fc8785, #fc8785 25px, #f4231f 25px, #f4231f 50px)",
+  },
+  dropzoneTextStyle: {
+    textAlign: "center",
+  },
+  uploadIconSize: {
+    width: 51,
+    height: 51,
+    color: "#909090",
+  },
+  dropzoneParagraph: {
+    marginBottom: 20,
+    marginTop: 20,
+  },
+};
+
+const snackbarAnchorOrigin = {
+  vertical: "bottom",
+  horizontal: "left",
+};
+
+/**
+ * This components creates a Material-UI Dropzone, with previews and snackbar notifications.
+ */
+const DropzoneAreaFunc = (props) => {
```

Comment:
Is there a need to suffix the Component with Func because its a Functional component?

Reply:
No, we should reuse the current naming.